index.html

```diff
@@ -107,7 +107,7 @@ <h3>Snacks</h3>
         </p>
         <div class="hidden"></div>
       </article>
-    <div id="food-table-info"></div>
+    <div id="diary-food-table-info"></div>
```

Comment:
Good addition to add specificity to which table we are appending to in diaryFoods.js